tools/github_actions_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ef
+
+echo 'python -m pytest --pyargs nilearn --cov=nilearn'
+python -m pytest --pyargs nilearn --cov=nilearn
```

Comment:
I think it would best not to run the tests from the directory in which the repo is checked out, but  from an empty (temporary) directory